PlayCover/en.lproj/Localizable.strings

```diff
@@ -20,6 +20,18 @@
 "playapp.noSources.title" = "No IPAs Installed";
 "playapp.noSources.subtitle" = "You currently have no IPAs installed. Click the button below to import one.";
 
+"ipaLibrary.info" = "App Info";
+"ipaLibrary.info.title" = "%@ Info";
+"ipaLibrary.info.notAvailable" = "Link not valid for this app";
+"ipaLibrary.info.tab.info" = "Info";
+"ipaLibrary.info.appName" = "App Name:";
+"ipaLibrary.info.bundleIdentifier" = "Bundle Identifier:";
```

Comment:
I suggest avoiding string duplicities in the localizations file by using some of those already defined for `settings.info.`.